profile/isa_ro_crate.md

```diff
@@ -248,9 +248,10 @@ It is based on [schema.org/ScholarlyArticle](https://schema.org/ScholarlyArticle
 |headline|MUST|Text||
 |author|SHOULD|[schema.org/Person](https://schema.org/Person)||
 |url|SHOULD|URL||
+|identifier|SHOULD|Text or URL or [schema.org/PropertyValue](https://schema.org/PropertyValue)|One or many identifiers for this article like a DOI or PubMedID. Can be of type PropertyValue to indicate the kind of reference.|
 |creativeWorkStatus|COULD|[schema.org/DefinedTerm](https://schema.org/DefinedTerm)|The status of the publication in terms of its stage in a lifecycle.|
 |disambiguatingDescription|COULD|Text|
```

Comment:
What is `disambiguatingDescription` referring to in this context?

Reply:
I don't remember exactly, but my guess would be the comments from ISA.

Reply:
Yes, you even mentioned it in Issue https://github.com/nfdi4plants/isa-ro-crate-profile/issues/3